AuthenticatedUserImplementation.md

```diff
@@ -0,0 +1,45 @@
+## Authenticated user implementation
```

Comment:
This approach is nice, but as another point of view, i prefer using a root controller, which manages session, all initialization(login)/termination(logout) processes by one point in the application. 

TODO: I will write a doc about this approach and put it here later, then we can discuss it, as well.

Reply:
It would be good to have both approaches documented here. I also think this `AuthenticatedUser` example should display `Codable` feature in Swift 4. Let's wrap this up.

Reply:
added